docs/assets/js/bibtex-unicorn.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Edit to the original bibtex_js_draw function
+ * Bibtex files need to be entered into individual <textarea>'s with class "reference"
+ * Generated content replaces the inputted bibtex data in each <textarea>
+ * Allows user to write notes/subheadings in between references
+ */
+$( document ).ready( function bibtex_js_draw_unicorn() {
+  var refs = document.querySelectorAll(".reference");
```

Comment:
Maybe restrict this selector to `textarea.reference`, if that's possible? That way we do not accidentally match non-`textarea` elements.

Reply:
yes, that works. I fixed it in the next commit.